### PR TITLE
add indices to speed up conseil

### PR DESCRIPTION
**Pull request by leenpaws**
Addresses issue #832 

Added the following indices to operations table
manager_pubkey
operation_group_hash
origination_contracts

and to account history table
account_id

This was done in response to the query taking more than 100ms and being the most frequent to be queried for the tezblock block explorer.

### CODE REVIEW

File: sql/conseil.sql [hunk 1]
```diff
@@ -585,6 +590,7 @@ CREATE INDEX endorsing_rights_level_idx ON tezos.endorsing_rights USING btree (l
 
 CREATE INDEX endorsing_rights_delegate_idx ON tezos.endorsing_rights USING btree (delegate);
 
+CREATE INDEX ix_delegate_slot ON tezos.endorsing_rights USING btree (delegate);
```
Comment (vishakh) [inline sql/conseil.sql:593]: There is no actual slot in the definition of the index.

Comment (leenpaws) [reply sql/conseil.sql]: Fixed.

File: conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/Tables.scala [hunk 2]
```diff
@@ -1751,6 +1757,9 @@ trait Tables {
 
     /** Index over (level) (database name endorsing_rights_level_idx) */
     val index2 = index("endorsing_rights_level_idx", level)
+
+    /** Index over (delegate) (database name ix_delegate_slot) */
+    val index3 = index("ix_delegate_slot", delegate)
```
Comment (ivanopagano) [inline conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/Tables.scala:1762]: the generated index name referred in the val doesn't match with the db-schema which is `ix_delegate`

File: sql/conseil.sql [hunk 3]
```diff
@@ -448,7 +450,9 @@ CREATE TABLE tezos.operations (
     utc_time character varying NOT NULL
 );
 
-
+ CREATE INDEX ix_manager_pubkey ON tezos.operations USING btree (manager_pubkey);
+ CREATE INDEX ix_operation_group_hash ON tezos.operations USING btree (operation_group_hash);
+ CREATE INDEX ix_originated_contracts ON tezos.operations USING btree (originated_contracts);
```
Comment (ivanopagano) [inline sql/conseil.sql:455]: I'm honestly wondering how the new indexes don't reflects on the PlatformDiscoveryOperations, that should mention those fields as UniqueKeys... just the same as with the other similar PR targeting the release tag branch...?

Yet the tests passes...

Comment (ivanopagano) [reply sql/conseil.sql:455]: 🤷‍♂️

Comment (ivanopagano) [reply sql/conseil.sql:455]: Man! I think I've commented the wrong PR, that's why I got inconsistent findings... 🤦

File: conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/Tables.scala (hunk 2 above)
Comment (ivanopagano) [reply conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/Tables.scala:1762]: the comment in the previous line should be amended accordingly, too

**Pull request merged**